frontend/src/components/Sidebar/RequirementSection.tsx

```diff
@@ -253,10 +256,19 @@ class RequirementSectionComponent extends React.Component<
    * Maps the given list of requirements to the render function.
    * @param reqs the list of requirements to be rendered
    */
-  parseRequirements(reqs: Requirement[]) {
+  parseRequirements(
+    reqs: Requirement[],
+    rootRequirementSection: ANDSection | ORSection
```

Comment:
Could you add `rootRequirementSection` to the param docstrings for `parseRequirements`, `renderRequirement`, and `convertTypeToText`?